donate/settings/__init__.py

```diff
@@ -7,7 +7,7 @@
 from .sentry import Sentry
 from .testing import Testing
 from .development import Development, ThunderbirdDevelopment
-from .review_app import ReviewApp
+from .review_app import ReviewApp, ThunderbirdReviewApp
```

Comment:
same as https://github.com/mozilla/donate-wagtail/pull/719 but necessary here in order to test things.